.github/workflows/ci.yaml

```diff
@@ -46,6 +46,7 @@ jobs:
         with:
           ruby-version: ${{ matrix.ruby }}
           bundler-cache: true
+          rubygems: latest
```

Comment:
I can use a conditional here to only use latest on 3.0

Reply:
I see the previous CI failure. I want to keep Ruby 3.0 support so this is good. I didn't know about this flag for the ruby/setup-ruby action. Thank you.